osquery/remote/transports/tls.cpp

```diff
@@ -24,7 +24,7 @@ namespace osquery {
 const std::string kTLSUserAgentBase = "osquery/";
 
 /// TLS server hostname.
-CLI_FLAG(string,
+FLAG(string,
```

Comment:
I'm leery of changing this

Reply:
Yes, please do not change this, it is required for configuration, chicken and egg problem so to speak.